ui/search.html

```diff
@@ -23,79 +23,111 @@
         <div data-widget="topnavigation"><!-- --></div>
```

Comment:
When the top navigation is used on the search page, the search bar in the top navigation should be hidden in favour of just the search icon button.